waku/v2/node/sqlite.nim

```diff
@@ -0,0 +1,207 @@
+import 
```

Comment:
Does this belong in the node package? Think we might be putting a bit too much in there

Reply:
im happy to move it elsewhere.